### DISCUSSION TITLE
[Question]: Remove model menu or hide models

### DISCUSSION THREAD

**Discussion by comeback01** [danny-avila/LibreChat#1719] [Q&A]
### What is your question?

Ideally I'd like to hide the "models" menu, but if I can hide the model names that's fine too.

I've modified the file /home/LibreChat/client/src/components/Input/ModelSelect/ModelSelect.tsx

this way 

```
 <OptionComponent
      conversation={conversation}
      setOption={setOption}
      models={models}
      showAbove={showAbove}
      popover={isMultiChat}

```
```
 <OptionComponent
      conversation={conversation}
      setOption={setOption}
      models={[]}
      showAbove={showAbove}
      popover={isMultiChat}
```
The model list is no longer selectable, but the default model name is still displayed.

Thanks for your help

### More Details

I've modified the file /home/LibreChat/client/src/components/Input/ModelSelect/ModelSelect.tsx

this way 

```
 <OptionComponent
      conversation={conversation}
      setOption={setOption}
      models={models}
      showAbove={showAbove}
      popover={isMultiChat}

```
```
 <OptionComponent
      conversation={conversation}
      setOption={setOption}
      models={[]}
      showAbove={showAbove}
      popover={isMultiChat}
```
The model list is no longer selectable, but the default model name is still displayed.

Thanks for your help

### Screenshots

![Capture d'écran 2023-05-14 140511](https://github.com/danny-avila/LibreChat/assets/17897116/1cd087ea-923a-4661-950d-dae5b4df7e62)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
You can try commenting out the component entirely. Your exact suggestion is planned but not yet implemented: https://github.com/danny-avila/LibreChat/discussions/1291

Introducing the `librechat.yaml` file was the first step towards this.
